mln/admin/__init__.py

```diff
@@ -60,6 +60,12 @@ class InventoryStackAdmin(StackAdmin):
 
 custom[InventoryStack] = InventoryStackAdmin
 
+class MessageTemplateAdmin(admin.ModelAdmin): 
+	"""Hide this from the UI, as all editing should happen in subclasses"""
+	def has_module_permission(self, request): return False
+
+custom [MessageTemplate] = MessageTemplateAdmin
+

```

Comment:
The usual way to handle this is to keep the admin view for the base class (`MessageTemplate`) but make the subclasses (`NetworkerReplyTrigger`, etc) inline to the base view (seen for instance with `make_inline(NetworkerMessageTrigger, NetworkerMessageAttachment`). This will effectively hide the subclasses from the main admin view but still allow them to be added/removed/edited in relation to an existing `MessageTemplate`. Your approach meanwhile would not allow a message template without a trigger to be added/edited as far as I can tell.

Reply:
> Your approach meanwhile would not allow a message template without a trigger to be added/edited as far as I can tell.

Correct, `MessageTemplate` has no use and I wanted to avoid the confusion of adding a `MessageTemplate` and nothing happening in-app, hence the comment. Hypothetically, even if we do use `MessageTemplate` for other purposes, it would still be better to create instances of those classes or inline `MessageTemplate`; I just don't see why editing a `MessageTemplate` directly would ever be useful.